frappe/core/doctype/data_import/importer.py

```diff
@@ -589,17 +590,22 @@ def _parse_doc(self, doctype, columns, values, parent_doc=None, table_df=None):
 		for key in frappe.model.default_fields + ("__islocal",):
 			doc.pop(key, None)
 
+		recordIsEmpty = True
```

Comment:
snake_case: `record_is_empty`

Reply:
Apologies for overlooking the naming style. Renamed variable.